tools/fixFormat.js

```diff
@@ -67,6 +67,21 @@ async.each(packages, function(item, callback) {
     pkg.licenses = pkg.license;
     delete pkg.license;
   }
+  if (pkg.npmFileMap) {
+    for (var i in pkg.npmFileMap) {
+      var npmbasePath = pkg.npmFileMap[i].basePath;
+      if (npmbasePath && npmbasePath.length != 0 && ((typeof npmbasePath) == "string")) {
+          npmbasePath = (npmbasePath).replace(/^\/|\/$/g , "");
```

Comment:
This pattern can not handle multiple slashes like `///dist///` which we don't want.